### PR TITLE
Add ESLint to Frontend Package

### PR DESCRIPTION
**Pull request by himanshukandpal-28**
Added ESLint to the frontend project and enforce the recommended rules, also added:

- npm run lint: Fix the fixable lint issues (including prettier)

- npm run lint:test: Just check if the code is well-formatted (used in the CI)

**Note** : Multiple files have changed after the linitng is done.

**Issue**: https://github.com/aerabi/events/issues/10

### CODE REVIEW

File: frontend/events/angular.json [hunk 1]
```diff
@@ -97,5 +97,8 @@
         }
       }
     }
+  },
+  "cli": {
+    "analytics": "744777e4-3f5f-4014-81af-1a09f415bf08"
```
Comment (aerabi) [inline frontend/events/angular.json:102]: Is this change necessary for the linter change?

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
```
Comment (aerabi) [inline .idea/.gitignore:2]: I would remove the `.idea` directory from the commit and add it to the `.gitignore` file. Could you please do that as well?

**Review by aerabi — COMMENTED** [review]
Awesome work! Thanks! I just left two comments. 🎉

**Comment by himanshukandpal-28** [comment]
Hey, sorry doing the change had to close the PR because it got submitted from my other account, doing the change and adding

**Comment by aerabi** [comment]
No worries, take your time. 👍
